app/index.sass

```diff
@@ -2,25 +2,12 @@
 // and any config relating to the build.
 // app config (like bootstrap variables) are also here, before anything else.
 
-$font-path: "../fonts";
-
-$icon-font-path: "#{$font-path}/bootstrap/";
-@import "bootstrap";
-
-$fa-font-path: "#{$font-path}/font-awesome";
-@import "font-awesome";
-
-@import "awesome-bootstrap-checkbox";
-
-@import "datetimepicker.css";
-@import "xeditable.css";
-
 // feedback form
 // NOTE where should this code be?
-.feedback {
-  z-index: 100000; // always on top
-  position: fixed;
-  right: round($grid-gutter-width / 2);
-  bottom: round($grid-gutter-width / 2);
-  background-color: #FFC0CB;
-}
+
+// ask mikey how to mass import shit
```

Comment:
there's no way that i'm aware of to mass import via sass, but we could change how our sass is built within the Gulpfile so that it concats all the sass files into a single file before running through libsass, if we wanted.